configure.py

```diff
@@ -1299,7 +1299,9 @@ def configure_zstd(build_dir, mode):
             restat = 1
             description = NINJA $out
         rule ragel
-            command = {ragel_exec} -G2 -o $out $in
+            # sed away a bug in ragel 7 that emits some extraneous _nfa* variables
+            # (the $$ is collapsed to a single one by ninja)
+            command = {ragel_exec} -G2 -o $out $in && sed -i -e '1h;2,$$H;$$!d;g' -re 's/static const char _nfa[^;]*;//g' $out
             description = RAGEL $out
```

Comment:
Please fold this into the commit that introduces the problem, so we don't break bisectability.

Reply:
@fastio please merge these two patches (unless I'm missing something, this appears to be Avi's last request), and I will commit the resulting version. Thanks.

Reply:
Hi @nyh, What do you mean is that the commit 5082a54 should be merged to the previous one (e9738a4) ?